### ISSUE TITLE
MNIST learning rate

### ISSUE THREAD

**Issue by glangford** [XifengGuo/CapsNet-Keras#35]
The paper says in 4:

> Our implementation is in TensorFlow (Abadi et al. [2016]) and we use the Adam optimizer (Kingma and Ba [2014]) *with its TensorFlow default parameters, including the exponentially decaying learning rate*

The TensorFlow defaults for Adam are described here:
<https://www.tensorflow.org/api_docs/python/tf/train/AdamOptimizer>

The current capsulenet.py uses `lr_decay` as a callback to modify the learning rate, but there isn't any evidence that the paper follows this method. Should the `lr_decay` callback be removed since Adam already decays the learning rate? 
(update: the TensorFlow and Keras defaults for Adam appear to be the same)

**Comment by XifengGuo** [comment]
@glangford As I said in README.md, I'm not sure if the paper used this learning rate decay method. I found that adopting `lr_decay` can lead to faster convergence. You can remove it and train for more epochs, it's your choice.
